package.json

```diff
@@ -11,6 +11,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.11.0",
     "react-addons-create-fragment": "^15.6.2",
+    "react-markdown": "^4.2.2",
```

Comment:
Can you do `npm install` and commit the `package-lock.json` file?

Reply:
@johnsaigle I think master's package-lock.json isn't correct currently.

Reply:
I'm getting the git status message `modified: package-lock.json` after running npm install on aces/master and when trying to checkout to another branch.

Reply:
@johnsaigle I committed the package-lock.json since the new npm module updates it. I was a little confused at first. 🙂

Reply:
It looks like the new dependency isn't in that file though. It's just changing a few instances of http to https.

Reply:
@johnsaigle I'm clueless to why react-markdown wasn't in package-lock.json after running npm install in the previous commit. The latest commit now has the react-markdown inside the package-lock.json. I had to delete the `node_modules` & `package-lock.json` before running npm install.